dsa-23au/java-dsa/datasets/src/test/java/dev/codewithfriends/NameCrawlerTest.java

```diff
@@ -0,0 +1,24 @@
+package dev.codewithfriends;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import java.util.Map;
+
+/**
+ * Unit test for scraping from Names database
+ */
+public class NameCrawlerTest
+{
+
+    @Test
+    public void testGetNamesByLetterPage()
+    {
+        NameCrawler.loadHeadersMap();
+        Map<String,String> meaningsMap = NameCrawler.getNamesByLetterPage('q', 1);
+        assertEquals(100, meaningsMap.size());
+        assertEquals("Night Blooming Flower", meaningsMap.get("Quynh"));
+    }
```

Comment:
This line is using an assertion to check if the size of the meaningsMap is equal to 100. It's verifying that the method has returned 100 names starting with 'q' on the first page.